cmd/integration_test/build/buildspecs/tinkerbell-test-eks-a-cli.yml

```diff
@@ -60,10 +61,14 @@ phases:
       - source ${CODEBUILD_SRC_DIR}/cmd/integration_test/build/script/create_infra_config.sh
       - ${CODEBUILD_SRC_DIR}/cmd/integration_test/build/script/start_docker.sh
       - export CLUSTER_NAME_PREFIX="${BRANCH_NAME//./-}"
-      - >
-        ./bin/test e2e cleanup vsphere
-        -n ${CLUSTER_NAME_PREFIX}
-        -v 4
+      - make build-integration-test-binary
+      - make build-eks-a-for-e2e
+      - make e2e-tests-binary E2E_TAGS="e2e tinkerbell" E2E_OUTPUT_FILE=bin/tinkerbell/e2e.test
+      - curl -s https://dev-release-prod-pdx.s3.us-west-2.amazonaws.com/bundle-release.yaml?versionId=7ZBFyTiC5_3RRfPKGvzMaA6DlmW2yExs -o bin/local-bundle-release.yaml
```

Comment:
Try without the version override? I think it's been fixed.

Reply:
I didn't try, but this versionId was newer than the one you used. It should be good